frame/dapps-staking/src/lib.rs

```diff
@@ -819,9 +814,22 @@ impl<T: Trait> Module<T> {
 
         let total_staked = Self::eras_total_stake(era);
 
+        let threshold = Self::eras_total_stake(&era) / BalanceOf::<T>::from(10);
```

Comment:
unused variables? Maybe, it's calculated in `compute_reward_for_nominator`.

Reply:
fixed in 1aab0b8